ros_bridge_interface/launch/interface.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" >
+     <arg name="port" value="9091"/>
```

Comment:
are these arguments default? Or will we need to change the port later?

Reply:
This is the port that is set in MovoConfig. I believe the default is 9090. I would like to make this value a parameter and controlled from one spot eventually but didn't have time yet.